sunpy/coordinates/wcs_utils.py

```diff
@@ -129,6 +129,7 @@ def solar_wcs_frame_mapping(wcs):
                              attrs[1] * u.deg,
                              attrs[2] * u.m,
                              **kwargs)
+            break
```

Comment:
Could you add a quick comment above where the `required_attrs` dictionary is defined that the order of entries matters there?